specification/security/resource-manager/Microsoft.Security/stable/2017-08-01/complianceResults.json

```diff
@@ -54,6 +54,9 @@
               "$ref": "../../../common/v1/types.json#/definitions/CloudError"
             }
           }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
```

Comment:
The response type `ComplianceResultList` doesn't contain a field named `nextLink`, it will need to be added.